### PR TITLE
Bump jenkins/inbound-agent from 4.11.2-4 to 3107.v665000b_51092-5

### PR DESCRIPTION
**Pull request by NotMyFault**

### CODE REVIEW

File: charts/jenkins/Chart.yaml [hunk 1]
```diff
@@ -33,6 +33,6 @@ annotations:
     - name: k8s-sidecar
       image: kiwigrid/k8s-sidecar:1.15.0
     - name: inbound-agent
-      image: jenkins/inbound-agent:4.11.2-4
+      image: jenkins/inbound-agent:latest
```
Comment (timja) [inline charts/jenkins/Chart.yaml:36]: why latest? wouldn't it be be better to fix the version and then have renovate / update cli update it?

Comment (NotMyFault) [reply charts/jenkins/Chart.yaml]: Right, good catch!

**Review by timja — APPROVED** [review]

File: charts/jenkins/unittests/jcasc-config-test.yaml [hunk 2]
```diff
@@ -65,7 +65,7 @@ tests:
                           - envVar:
                               key: "JENKINS_URL"
                               value: "http://RELEASE-NAME-jenkins.default.svc.cluster.local:8080/"
-                        image: "jenkins/inbound-agent:4.11.2-4"
+                        image: "jenkins/inbound-agent:3107.v665000b_51092-5"
```
Comment (timja) [inline charts/jenkins/unittests/jcasc-config-test.yaml:68]: this image doesn't exist:

```
docker pull jenkins/inbound-agent:3107.v665000b_51092-5
Error response from daemon: manifest for jenkins/inbound-agent:3107.v665000b_51092-5 not found: manifest unknown: manifest unknown
```

I keep getting emails from artifacthub about it failing to scan it

Comment (NotMyFault) [reply charts/jenkins/unittests/jcasc-config-test.yaml:68]: Right, looks like something changed within the deployment of these images. There's no image similar to `4.11.2-4` left in https://hub.docker.com/r/jenkins/inbound-agent/tags?page=1&name=3107.v665000b_51092-5 👀

Comment (timja) [reply charts/jenkins/unittests/jcasc-config-test.yaml:68]: at a guess `-5` failed on linux publishing, but windows passed. I can see `-4`

**Pull request merged**